src/command_modules/azure-cli-vm/azure/cli/command_modules/vm/_operation_factory.py

```diff
@@ -0,0 +1,365 @@
+#---------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+#---------------------------------------------------------------------------------------------
+
+## SDK COMMAND OPERATIONS
```

Comment:
I had a prototype that did use strings with imports.
e.g. 
```
cli_command('my command', lambda: import_module('azure.mgmt.compute.operation', 'create_or_update'))
```

Issue with this is:
- import syntax changes as you can't have the `from X import Y`.
- IntelliSense/PyLint would not be able to run any checks or help as our imports become strings.